src/tinuous/circleci.py

```diff
@@ -68,8 +68,18 @@ def get_pipelines(self) -> Iterator[Pipeline]:
         for item in self.paginate(f"/v2/project/gh/{self.repo}/pipeline"):
             yield Pipeline.model_validate(item)
 
-    def get_workflows(self, pipeline_id: str) -> Iterator[Workflow]:
+    def get_workflows(
+        self, pipeline_id: str, pipeline_number: int
+    ) -> Iterator[Workflow]:
         for item in self.paginate(f"/v2/pipeline/{pipeline_id}/workflow"):
+            if item.get("id") is None:
```

Comment:
and without `id` this workflow is "no good", as there is nothing else worth checking to guarantee that we do not miss any workflow? (may be the `name` and `status`?)

Reply:
@yarikoptic Without the workflow ID, we can't get any information about jobs, and it's the jobs that have the actual logs.